package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
I'm not sure either.  I was just going by this statement on the wiki, which also isn't entirely clear:

> jsnext:main will point to a module that has ES2015 module syntax but otherwise only syntax features that node supports.

"syntax features that node supports", but which version of node?  It seems reasonable to me to assume some lowest common denominator, like ES5.